appengine_config.py

```diff
@@ -97,7 +97,7 @@ def save(self):
     os.path.join(ROOT_PATH, 'third_party', 'graphy-1.0.0'),
     os.path.join(ROOT_PATH, 'third_party', 'html5lib-python-1.0.1'),
     os.path.join(ROOT_PATH, 'third_party', 'mutagen-1.38'),
-    os.path.join(ROOT_PATH, 'third_party', 'requests-2.10.0'),
+    os.path.join(ROOT_PATH, 'third_party', 'requests-2.21.0'),
```

Comment:
The latest version of requests shown [here](https://2.python-requests.org/en/master/community/updates/#id1) is 2.22. We should be upgrading to 2.22 then.

Reply:
Hi @apb7, yes 2.22 has officially stopped support for Python 3.4 as their was an onling project for migrating the code to python 3 i was not sure so upgrade it to the latest feasible verison

Reply:
No, I think we should be going for 2.22 since it supports Python 2.7 and >=3.5. At the moment, we're only making the code base easier to migrate to Python 3 (and not any specific version).

Reply:
Sounds good to me, I've updated the version and the changes are trivial. Thanks!

Reply:
The tests are failing with error
```
Parse error at third_party/generated/js/third_party.js:21644,12
import * as util from './util';
            ^
ERROR: Unexpected token: name «util», expected: punc «;»
```
I'll try to debug Thank!

Reply:
Hi @apb7,  The above failing tests has been resolved Thanks!